packages/document/module-doc/docs/en/components/register-esbuild-plugin.mdx

```diff
@@ -5,7 +5,7 @@ export default defineConfig({
   buildConfig: {
     esbuildOptions: options => {
       options.plugins = [myEsbuildPlugin, ...options.plugins];
-      return option;
+      return options;
```

Comment:
how about `packages/document/module-doc/docs/zh/components/register-esbuild-plugin.mdx`

Reply:
It has been fixed too.